root/static/scripts/edit/externalLinks.js

```diff
@@ -652,6 +652,7 @@ const URL_SHORTENERS = [
   'tiny.cc',
   'tinyurl.com',
   'tourlink.to',
+  'trac.co', // Host links can be legitimate, but non-root are aggregators
```

Comment:
Disambiguate `non-root` thing in comment:

```suggestion
  'trac.co', // Host links can be legitimate, but non-root paths are aggregators
```